Microsoft.NET.Build.Containers/CreateNewImage.cs

```diff
@@ -95,72 +95,34 @@ public override bool Execute()
             return !Log.HasLoggedErrors;
         }
 
-        Registry reg;
```

Comment:
This is going to be a bit of an ugly merge with some of @baronfel's PRs. I'll let y'all work out the order.

Reply:
I should strip this out and place it in the last PR that will bring in the VS changes.